mozjs-sys/build.rs

```diff
@@ -758,6 +758,9 @@ impl BuildTarget {
                 // is more than bindgen can cope with.
                 "JS::Rooted",
                 // We don't need them and bindgen doesn't like them.
+                "JS::StackGCVector.*",
+                "JS::RootedVector_Vec",
+                "JS::RootedVector_Base",
```

Comment:
The comment is wrong for those types, beacuse we obviously need them. Maybe they should be in their own section.